src/sage/groups/perm_gps/permgroup.py

```diff
@@ -2356,7 +2373,7 @@ def order(self):
             sage: G.order()
             1
 
-        :meth:`cardinality` is just an alias::
+        :meth:`cardinality` is an alias::
```

Comment:
This is no longer true with this change. Although I still don't see why you need to change how this is done with defining `order` and then making `cardinality` an alias. There is no harm in having it IMO.